### PR TITLE
Add SSR-Speech as lib for download stats

### PR DESCRIPTION
**Pull request by WangHelin1997**
Hi, I want to add the model [SSR-Speech](https://github.com/WangHelin1997/SSR-Speech) as a library for download stats. Thanks!
The huggingface repos are [https://huggingface.co/westbrook/SSR-Speech-Mandarin](https://huggingface.co/westbrook/SSR-Speech-Mandarin) and [https://huggingface.co/westbrook/SSR-Speech-English](https://huggingface.co/westbrook/SSR-Speech-English)

### CODE REVIEW

**Comment by julien-c** [comment]
can you update the PR's title and description @WangHelin1997? Thanks!

**Review by Vaibhavs10 — COMMENTED** [review]
Hi @WangHelin1997 - Thanks for your PR, as mentioned above please update the Title and add a simple description.

On a quick review, first of all I opened a PR on your English model repo: https://huggingface.co/westbrook/SSR-Speech-English/discussions/1

This PR will make allow the model downloads to be tracked actually, we need the repos to be associated with the tag (in this case `ssr-speech`) - this will be handy when you release your Mandarin checkpoint as well.

Next, please make sure that the Hub model repo has a well documented README: https://huggingface.co/westbrook/SSR-Speech-English (you can copy information from your github repo as well)

It might be worth adding a model snippet here as well, ref: https://github.com/huggingface/huggingface.js/blob/main/packages/tasks/src/model-libraries-snippets.ts

File: packages/tasks/src/model-libraries.ts
Comment (Wauplin) [inline packages/tasks/src/model-libraries.ts:694]:
```suggestion
		countDownloads: `path_extension:".pth"`,
```
Let's count the downloads only on 1 file per repo (here English.pth and Mandarin.pth). Since both the .pth and the .th files seem to be needed to load the model, counting downloads on both would count everything double

File: packages/tasks/src/model-libraries.ts
Comment (Wauplin) [inline packages/tasks/src/model-libraries.ts:689]: Can you update both model card ([here](https://huggingface.co/westbrook/SSR-Speech-Mandarin/blob/main/README.md) and [here](https://huggingface.co/westbrook/SSR-Speech-English/blob/main/README.md)) to add `library_name: ssr-speech` in the model card metadata? This way there will be at least 2 models listed in https://huggingface.co/models?other=ssr-speech before we merge this PR.

Comment (Wauplin) [inline packages/tasks/src/model-libraries.ts:689]: Can you move the `ssr-speech` entry up in the file? We try to keep alphabetical order in the libraries definition.

**Review by Wauplin — COMMENTED** [review]
Hi @WangHelin1997, thanks for clarifying the PR! I've left a few comments, after that we should be good to merge!

**Review by WangHelin1997 — COMMENTED** [review]
move the ssr-speech entry up in the file to keep alphabetical order

**Review by Wauplin — APPROVED** [review]
Thanks for the changes @WangHelin1997! Everything looks good to me now :hugs: 

I'll let @Vaibhavs10 confirm and merge the PR :)

**Review by Vaibhavs10 — APPROVED** [review]
LGTM, thanks a lot for iterating on this! ❤️

**Pull request merged**
